MiniApp/Classes/Protocols/MiniAppAuthenticationDelegate.swift

```diff
@@ -0,0 +1,24 @@
+/**
+ Public Protocol that will be used by the Mini App to get Authentication
+ info, such as Access token
+*/
+public protocol MiniAppAuthenticationDelegate: class {
```

Comment:
For Android, we added this to `UserInfoBridgeDispatcher`. So maybe the `MiniAppUserInfoDelegate` is a better place for this? This is related to user info, so I think it makes more sense.

Reply:
I thought having it as a separate delegate would be better. As of now, we retrieve only token and expiryDate and I thought we will have more into R SDK in the future. No worries, updated 👍